docs/topics/assets/plantuml/README.md

```diff
@@ -0,0 +1,41 @@
+<!---
+  Copyright (c) 2021 Robert Bosch GmbH
```

Comment:
Please change to "Bosch.IO GmbH" for the IoT-Event-Analytics Repository

Reply:
Done